src/xdocs/releasenotes.xml

```diff
@@ -16,8 +16,8 @@
   <body>
     <!-- placeholder for a new section -->
 
-    <div class="releaseDate">25.02.2019</div>
     <section name="Release 8.18">
+      <div class="releaseDate">25.02.2019</div>
```

Comment:
This is a generated section and cannot be modified easily.
Please explain why this change is needed, and if it is valid, it will also have to be updated at https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/src/main/resources/com/github/checkstyle/templates/xdoc_freemarker.template#L2 .

Reply:
The change is necessary, because the only valid content of `<body>` tag is `<section>`, so the `<div>` cannot be before the section which is why I put it inside and moved it to the proper place via CSS.
I was not aware that this file is generated, I now also adapted the freemarker template and the tests, the generation produces the proper result here now, I'll create a PR to the `contribution` repo when we agreed here.

Reply:
@romani ping

Reply:
Just to mention it again, if you are not ok with this, you can never enable the validation option of the maven-site-plugin, as the `<div>` as child of the `<body>` is invalid and thus will not validate.
Up to now the validation option was disabled due to a bug linked in the `pom.xml` or rather an underlying bug in the `maven-xml-plugin` that since was fixed.

Reply:
Can you provide screenshot of how page will look with your changes? If it is no difference, then I am fine.
I would like @romani to confirm before any changes are made to contribution.

Reply:
@Vampire , please share a screenshot that it renders fine, command to generate site is `mvn site -Pno-validations`.

Reply:
It will look exactly like currently, one pixel give or take.

Reply:
http://rveach.no-ip.org/checkstyle/regression/reports/fix-invalid-doc/releasenotes.html

Output looks the same. When PR is ready to be merged, we will have to create PR for other repo.
I will this as unresolved to remind us.

Reply:
I have the respective PR commit for PR ready already, I could open PR to the other repo as reminder if you want.